test/views/EditSetViewSpec.scala

```diff
@@ -22,22 +22,26 @@
 package views
 
 import org.jsoup.nodes.Document
+import play.api.test.Helpers.{ contentType, defaultAwaitTimeout }
 import play.api.test.{ CSRFTokenHelper, FakeRequest, Helpers }
 import play.twirl.api.Html
-import support.BaseSpec
+import support.BaseViewSpec
 import support.CommonMatchers._
 import uk.gov.hmrc.govukfrontend.views.Aliases.Pagination
-import uk.gov.nationalarchives.omega.editorial.models.{ EditSet, EditSetEntry }
-import uk.gov.nationalarchives.omega.editorial.services.EditSetPagination.EditSetPage
+import uk.gov.hmrc.govukfrontend.views.html.components.{ GovukPagination, GovukTable }
+import uk.gov.hmrc.govukfrontend.views.viewmodels.table.Table
+import uk.gov.nationalarchives.omega.editorial.models.{ EditSet, EditSetEntry, User }
 import uk.gov.nationalarchives.omega.editorial.services.EditSetEntryRowOrder
+import uk.gov.nationalarchives.omega.editorial.services.EditSetPagination.EditSetPage
 import uk.gov.nationalarchives.omega.editorial.views.html.editSet
 
-class EditSetViewSpec extends BaseSpec {
+class EditSetViewSpec extends BaseViewSpec {
 
   "Edit set Html" should {
     "render the given title and heading" in {
-
-      val editSetInstance = inject[editSet]
+      val mockGovukPagination = mock[GovukPagination]
```

Comment:
Does these have to be mocked? 

If it can be easily constructed, I'd say that's always preferable to a mock.

Reply:
Replaced mock objects with govuk component.

Reply:
Confirmed.

Reply:
However, in some case the val name still has a `mock` prefix.

Reply:
Well spotted, done.